libs/community/langchain_community/document_loaders/parsers/audio.py

```diff
@@ -298,9 +298,16 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
             if self.base_url:
                 openai.api_base = self.base_url
 
-        # Audio file from disk
+        # get the audio
+        if isinstance(blob.data, bytes):
```

Comment:
could you factor this out into a utility to avoid the same check in several places?

Reply:
no need for a separate file, what you added is exactly what i was thinking. thanks!